app/templates/gulpfile.js

```diff
@@ -131,7 +130,12 @@ gulp.task('wiredep', function () {
 });
 
 gulp.task('build', ['jshint', 'html', 'images', 'fonts', 'extras'], function () {
-  return gulp.src('dist/**/*').pipe($.size({title: 'build', gzip: true}));
+  return gulp.src('dist/**/*', {base: 'dist'})
+    .pipe($.revAll({ignore: ['favicon.ico', 'apple-touch-icon.png', 'robots.txt', '.html']}))
```

Comment:
https://github.com/smysnk/gulp-rev-all/blob/591d18d4247f287b0821896f8a266b6f7128963c/README.md#upgrading-to-v08x

>New Feature: Ignoring files has changed, ignore option has been removed and has been replaced with dontGlobal, dontRenameFile, dontUpdateReference, dontSearchFile which allows for more control and less ambiguity on what is being ignored